docs/development/Running_Acceptance_Tests.md

```diff
@@ -33,20 +34,19 @@ so that the acceptance can be run from source.
 ## Travis and Jenkins
 
 docker-compose is used to start the required components in the Travis and Jenkins Workspaces to avoid any
-conflicts with port numbers.  These steps can be run locally:
+conflicts with port numbers.  These steps can be run locally.
 
-to start these containers:
+To start these containers:
 ```
 docker-compose up [-d]
```

Comment:
I don't really like code snippets that you can't actually copy and paste - the intention here is to indicate that `-d` is optional. The `-d` flag is a normal docker-compose option so it is well documented outside our work so I think ignore it or add it but don't add the square braces.

Reply:
agreed, I've removed `[-d]`